core/domain/exp_domain.py

```diff
@@ -2980,21 +2980,18 @@ def get_state_names_mapping(self, change_list):
             dict. The state_names_mapping dict.
         """
         state_names_mapping = {}
```

Comment:
The var names could be clearer; it's hard to puzzle out what's going on. Call this old_to_new_state_names, and call the var below new_to_old_state_names.